### ISSUE TITLE
build_skinning_matrix output size

### ISSUE THREAD

**Issue by msnhao** [dilevin/CSC417-a3-finite-elements-3d#11]
In build_skinning_matrix.h, the comments say

//  V_skin - lx3 matrix of vertices of the display mesh
//Output:
//  N - the lxn sparse skinning matrix 

Since there are 'l' skin mesh vertices, shouldn't N be sized (3*l)x(3*n)? Since there are 3 coordinates per vertex?
That's what it looks like in the lecture 5 video as well.

**Comment by dilevin** [comment]
yes, the comments have a typo (I think I mentioned in class I miss the threes sometimes :) )

**Comment by abhimadan** [comment]
The comment on that file is actually correct - please make your skinning matrix with dimensions `l x n`. It's used in `visualization.cpp`, where you multiply the skinning matrix by a `n x 3` matrix.
